pkg/image/decompress.go

```diff
@@ -8,30 +8,34 @@ import (
 	"io/ioutil"
 
 	"github.com/golang/glog"
+	. "github.com/kubevirt/containerized-data-importer/pkg/common"
 	"github.com/xi2/xz"
 )
 
 func GzReader(r io.ReadCloser) (io.ReadCloser, error) {
-	glog.Infoln("GzReader: gz format")
-	return gzip.NewReader(r)
+	gz, err := gzip.NewReader(r)
+	if err != nil {
+		return nil, fmt.Errorf("GzReader: error creating gz Reader: %v\n", err)
+	}
+	glog.V(Vadmin).Infof("gzip: extracting %q\n", gz.Name)
+	return gz, nil
 }
 
 func XzReader(r io.ReadCloser) (io.ReadCloser, error) {
-	glog.Infoln("XzReader: xz format")
-	rdr, err := xz.NewReader(r, 0) //note: default dict size may be too small
+	glog.V(Vdebug).Infoln("XzReader: xz format")
```

Comment:
This line isn't symmetric with GzReader or TarReader

Reply:
True. I don't see in the xz reader (or hdr) where to get the compressed file name(s). Any pointers?

Reply:
oh, yes, fair point.  I don't think our xz lib has that capability